src/App.js

```diff
@@ -36,20 +38,51 @@ function App() {
     // Hope this helps :)
     //
     <>
-      <Navbar></Navbar>
-      <Routes> 
-        <Route path="/" element={<Home />} /> 
-        <Route path="/updates" element={<Updates />} /> 
-        <Route path="/about" element={<About />} /> 
-        <Route path="/contact" element={<Contact />} /> 
-        <Route path="/account" element={<Account />} /> 
-        <Route path="/project" element={<Project />} /> 
-        <Route path="/login" element={<Login />} />
-        <Route path="/signup" element={<Signup />} />
-        <Route path="/games" element={<Games />} />
-      </Routes>
-    </>
+
+      <FamilyList flashcards={flashcards} /><>
+        <Navbar></Navbar>
+        <Routes>
+          <Route path="/" element={<Home />} />
+          <Route path="/updates" element={<Updates />} />
+          <Route path="/about" element={<About />} />
+          <Route path="/contact" element={<Contact />} />
+          <Route path="/account" element={<Account />} />
+          <Route path="/login" element={<Login />} />
+          <Route path="/signup" element={<Signup />} />
+          <Route path="/games" element={<Games />} />
+        </Routes>
+      </></>
+
   );
 }
 
+const SAMPLE_FLASHCARDS = [
```

Comment:
I understand this is probably just here while you're testing new development of FFC, but in order to keep the main branch clean with "production-ready" code, I'm gonna ask if you can move this and SAMPLE_FLASHCARDS2 into a different file. we want to keep our root file (app.js) from having a bunch of component specific logic in it. any family flashcards related logic should remain in the family flashcards component/components if possible.

Reply:
Everything related to cards have been removed